Sources/SwifterSwift/UIKit/UITextFieldExtensions.swift

```diff
@@ -172,6 +172,19 @@ public extension UITextField {
         rightView = iconView
         rightViewMode = .always
     }
+    
+    /// Add tool bar to the textfield input accessory view.
+    /// - Parameters:
+    ///   - title: Pass your button title e.g "Done"
+    ///   - target: The object that receives the action message.
+    ///   - selector: Pass selector method to received bar button tapped from toolbar.
+    func addInputAccessoryView(title: String, target: Any, selector: Selector? = nil) {
```

Comment:
This isn't really generic enough. You're specifically just creating a toolbar to the keyboard with a button on the right with the given values. Maybe this function should be called, `addToolbarWithButton`?

You could also create a more generic function with a list of `UIBarButtonItem`s.
```Swift
func addToolbar(items: [UIBarButtonItem]?, height: CGFloat = 44) -> UIToolbar {
    let toolBar = UIToolbar(frame: CGRect(x: 0.0, y: 0.0, width: UIScreen.main.bounds.size.width, height: height))
    toolBar.setItems(items, animated: false)
    inputAccessoryView = toolBar
    return toolBar
}
```

Reply:
It makes sense to make a generic function with list of `UIBarButtonItems` instead of right aligned bar button. Thanks @guykogus  for the feedback. Addressed this changes